configs/98netflix

```diff
@@ -0,0 +1,6 @@
+#!/system/bin/sh
```

Comment:
We can replace patches and custom WebView with a lot of stuff in device tree, like on Galaxy S2 common device tree: https://github.com/CyanogenMod/android_device_samsung_galaxys2-common